docs/start/smesher/trusted_state_quicksync.md

```diff
@@ -0,0 +1,45 @@
+---
+id: trusted_state_quicksync
+title: Trusted State aka Quicksync
+---
+
+## Quicksync
+
+In response to the growing difficulty of syncing a fresh node, we have prepared a way to speed up the syncing process. Introducing, Quicksync.
+
+Before reading any further, plaese read the annoucement about Quicksync [here](https://spacemesh.io/blog/making-sync-faster/).
+
+### Getting the archive automatically
+
+#### Smapp
+
+Smapp will propose to run Quicksync automatically.
+
+#### CLI
+
+We have prepared a simple tool called [quicksync-rs](https://github.com/spacemeshos/quicksync-rs) that simplifies the process of getting the archive.
+
+### Getting the archive manually
+
+The most recent archive for a given version can be found at: `https://quicksync.spacemesh.io/<version>/latest.zst`. This URL will always redirect to the most recent archive for the given version.
+
+The archive is compressed with `zstd` and can be decompressed using the following command: `zstd -d --long=31 o state.sql latest.sql.zst`. Note that the decompressed archive should be placed in the node data directory of the node.
```

Comment:
Given that many users are working on Windows, it would be nice to include a Windows-friendly instruction too.

Reply:
Agree. As I see it, the `zstd` tool isn't available on windows. What should we do in this case?

Reply:
https://github.com/facebook/zstd/releases/tag/v1.5.6 Available for Windows.

Reply:
It seems so: https://github.com/facebook/zstd/releases/tag/v1.5.6